src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/JsonConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.config;
+
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
+import com.github.fge.jsonschema.main.JsonSchema;
+import com.github.fge.jsonschema.main.JsonSchemaFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+@Configuration
+public class JsonConfiguration {
+
+    @Bean
+    public ObjectMapper defaultObjectMapper() {
+        ObjectMapper mapper = new ObjectMapper();
+
+        mapper.enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);
```

Comment:
Will this also affect the features used in REST API as well, or just parsing json files in zip packages?

Reply:
Doesn't seem to take affect even with application config present:
```yaml
spring:
  jackson:
    mapper:
      ACCEPT_CASE_INSENSITIVE_ENUMS: true
```
suggests REST request bodies are deserialised in some other way :/

Reply:
This bean definition would override any jackson setting in config file I'm afraid.
It also defines `ObjectMapper` for entire application.

Perhaps it would be safer to either used a named bean or wrap parsing metadata into it's own class that uses Jackson internally?

Reply:
`ObjectMapper` as a bean is not interfering with any `Spring`'s internal processes. So yes - it is for entire application whenever we choose to inject but it will not be used nor overridden. `Spring` injects / uses new instance of `ObjectMapper` on its' own for every `REST` request.

This gives the freedom of not breaking the framework and still using `ObjectMapper` as a `Bean` for your application. We still would require same mapper feature everywhere

Reply:
If you define a bean in autoconfiguration it will be used by Spring boot. If you define it as a bean then Spring boot will use it when mapping to Java Object in REST API. So I would say let's separate this out into a separate class method as suggested by @rkondratowicz earlier. Also regarding above comment `spring.jackson.mapper.accept case_insensitive enums` it has to be all lower case for spring to consider config changes.

Reply:
ah i felt something off about this.

i'm leaning towards adding the config though, but if it doesn't work - will just leave up to the component to create object mapper.

it is safe to merge as nothing actually ground-breaking about this set up but will introduce a separate pr addressing this issue